cg/meta/deliver.py

```diff
@@ -277,3 +280,28 @@ def create_delivery_dir_path(self, case_name: str = None, sample_name: str = Non
             delivery_path = delivery_path / sample_name
 
         return delivery_path
+
+    @staticmethod
+    def get_delivery_arguments(case_obj: models.Family) -> Set[str]:
+        """Translates the case data_delivery field to pipeline specific arguments."""
+        delivery_arguments: Set[str] = set()
+        requested_deliveries: List[str] = re.split("[-_]", case_obj.data_delivery)
+        if DataDelivery.FASTQ in requested_deliveries:
+            delivery_arguments.add(Pipeline.FASTQ)
+        if DataDelivery.ANALYSIS_FILES in requested_deliveries:
+            delivery_arguments.add(case_obj.data_analysis)
```

Comment:
I think can be more future proof by something like:
```
       delivery_per_pipeline_map: Dict[str, str] = {DataDelivery.FASTQ: Pipeline.FASTQ,
                                                     DataDelivery.ANALYSIS_FILES: case_obj.data_analysis}
        for data_delivery, pipeline in delivery_per_pipeline_map.items():
            if data_delivery in requested_deliveries:
                delivery_arguments.add(pipeline)
```

Reply:
Nice 👍